.devcontainer/devcontainer.json

```diff
@@ -22,7 +22,7 @@
 	// "forwardPorts": [],
 
 	// Use 'postCreateCommand' to run commands after the container is created.
-	// "postCreateCommand": "yarn install",
```

Comment:
Unrelated change I tacked on because it's annoying that I have to install deps in every codespace (and that means that my ESLint and Prettier plugins don't work initially)